junit-platform-commons/src/main/java/org/junit/platform/commons/util/ReflectionUtils.java

```diff
@@ -565,7 +566,7 @@ static boolean isWideningConversion(Class<?> sourceType, Class<?> targetType) {
 	 * @return the corresponding wrapper type or {@code null} if the
 	 * supplied type is {@code null} or not a primitive type
 	 */
-	public static Class<?> getWrapperType(Class<?> type) {
+	public static @Nullable Class<?> getWrapperType(Class<?> type) {
```

Comment:
@marcphilipp given that the Javadoc mentions:

```
* @return the corresponding wrapper type or {@code null} if the
* supplied type is {@code null} or not a primitive type
```

should the parameter also have `@Nullable`?

```java
public static @Nullable Class<?> getWrapperType(@Nullable Class<?> type)
```

(I spotted it during the rebase of #4219)

After a quick look, I think there are no unit tests for `getWrapperType()` in `ReflectionUtilsTests`.

Happy to submit a PR to improve both topics, if you agree 🙂

Reply:
From my branch:

https://github.com/junit-team/junit5/blob/791778e4189e9e19a7908500b49113d21d20014e/junit-platform-commons/src/main/java/org/junit/platform/commons/support/conversion/TypeDescriptor.java#L61-L64

triggered this, which is somehow related to the following on `main`:

https://github.com/junit-team/junit5/blob/261c2f5a1576ea29add753b92499e7cf29342f23/junit-platform-commons/src/main/java/org/junit/platform/commons/support/conversion/ConversionSupport.java#L117

I try to figure out why the warning doesn't happen on `main` 🤔